public/partials/dashboardv2/edit.html

```diff
@@ -66,10 +66,21 @@ <h3 class="pull-left" editable-text="selectedDashboard.dashboardName">{{ selecte
     <section  class="content">
 
         <section id="settingsTab" class="results infinite ng-hide" ng-show="tabs.selected == 'settings'">
+
             <section class="layer scrollPane">
                 <div class="layer-inner-reports">
 
-                    <app-inspector element="selectedElement" properties="selectedReport.properties" dashboard="selectedDashboard" on-change="onChangeElementProperties"></app-inspector>
+                    <div id="limit-panel">
+                        <span style="color:#fff;" translate>Quick display with</span>
+                        <input type="number" ng-model="selectedDashboardLimit.value" size="8">
+                        <span style="color:#fff;" translate>results.</span>
+                        <button style="padding: 3px 10px;" ng-click="Redraw();" class="btn btn-default">
+                          <i class="fa fa-refresh hand-cursor"></i>
+                          <span translate>Redraw</span>
+                        </button>
+                    </div>
+
+                    <wst-inspector element="selectedElement" properties="selectedReport.properties" dashboard="selectedDashboard" on-change="onChangeElementProperties"></wst-inspector>
```

Comment:
I suspect a bad conflict resolution here.
wst-inspector is now named app-inspector